modules/tinymce/src/core/main/ts/undo/Setup.ts

```diff
@@ -12,6 +12,18 @@ import * as Levels from './Levels';
 import { endTyping, setTyping } from './TypingState';
 import { Locks, UndoLevel, UndoManager } from './UndoManagerTypes';
 
+const skipCommand = (cmd: string): boolean => {
```

Comment:
```suggestion
const ignoreCommand = (cmd: string): boolean => {
```
This would probably be more accurate as to what it's doing. Another option would be `shouldIgnoreCommand`.